src/App.js

```diff
@@ -46,14 +47,12 @@ export default function App() {
       filteredStoreItems = getFruit(initialStoreItems)
     }
     sortById(filteredStoreItems)
-  }
-  else if (filterTwo === 'alphabetically') {
+  } else if (filterTwo === 'alphabetically') {
     if (filter === 'veg') {
       filteredStoreItems = getVeg(initialStoreItems)
```

Comment:
There's a lot of if statements here. Could they be refactored if your rendering of the store items happened from a state variable that changed depending on the filter, so first filter is type, which changes the state array, then that is what is looked at when you change the sort by filter, so it works no matter what you change, and it will update every time you change, so there might be no need to do all this checking.